packages/microcosm-www-next/src/pages/index.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
+import debounce from 'lodash.debounce'
```

Comment:
just had someone on this other project load all of lodash just for the debounce...fun fact.

Reply:
tbh I did that initially before realizing error of my ways 😅